extension/src/popup/views/SendPayment/index.tsx

```diff
@@ -10,26 +10,25 @@ import { SendSettings } from "popup/components/sendPayment/SendSettings";
 import { SendSettingsFee } from "popup/components/sendPayment/SendSettings/TransactionFee";
 import { SendConfirm } from "popup/components/sendPayment/SendConfirm";
 
-// TODO - enforce can't move to next route data not given
 export const SendPayment = () => (
   <Switch>
     <PrivateKeyRoute exact path={ROUTES.sendPayment}>
       <Redirect to={ROUTES.sendPaymentTo} />
     </PrivateKeyRoute>
     <PrivateKeyRoute exact path={ROUTES.sendPaymentTo}>
-      <SendTo />
+      <SendTo previous={ROUTES.account} />
     </PrivateKeyRoute>
     <PrivateKeyRoute exact path={ROUTES.sendPaymentAmount}>
-      <SendAmount />
+      <SendAmount previous={ROUTES.sendPaymentTo} />
     </PrivateKeyRoute>
     <PrivateKeyRoute exact path={ROUTES.sendPaymentSettings}>
-      <SendSettings />
+      <SendSettings previous={ROUTES.sendPaymentAmount} />
     </PrivateKeyRoute>
```

Comment:
I may be missing something, but the Back Button should default to history.goBack (meaning, basically just press back in the browser window). Is that not sufficient? Looks like you're moving linearly through these screens

I see that going back on `sendPaymentConfirm` should take you back 2 screens to `sendPaymentSettings`, so that custom `previous` makes sense to me

Reply:
it's really just for the edge case of if you're in the middle of the sendPayment flow and your session times out so get redirected to `/unlock-account`. So after logging in goBack will take you back to `/unlock-account`